lib/pagination/paginate.js

```diff
@@ -0,0 +1,32 @@
+const Adapter = require('./adapter');
+const config = require('./config');
+const logger = require('../../test/app/logger/index');
+
+const page = options => (options.page === undefined ? config.default_page : options.page);
```

Comment:
You can do `const page = options => options.page || config.default_value`

Reply:
I had it like that before, but the problem that comes up is that if options.page is 0, it takes it as a "false" and puts the `config.default_value` value, which is not the expected behaviour: when page is 0, it should raise an exception, with that sentence that doesn't happen

Reply:
You are right